test/emmy/expression/cse_test.cljc

```diff
@@ -60,29 +60,3 @@
     (is (= [slimmed expected-subs]
            (c/extract-common-subexpressions expr vector opts))
         "subexpressions are again extracted in order.")))
-
-(def letsym
```

Comment:
In the future, I now know that we can do this by writing `let, which will resolve correctly to the namespaced symbol.

Reply:
gulp. I didn't even think of that, but that's indeed how it works!